src/JWT.ts

```diff
@@ -375,29 +346,12 @@ export async function createMultisignatureJWT(
 ): Promise<string> {
   if (issuers.length === 0) throw new Error('invalid_argument: must provide one or more issuers')
 
-  const timestamps: Partial<JWTPayload> = {
-    iat: Math.floor(Date.now() / 1000),
-    exp: undefined,
-  }
-  if (expiresIn) {
-    if (typeof expiresIn === 'number') {
-      timestamps.exp = <number>(payload.nbf || timestamps.iat) + Math.floor(expiresIn)
-    } else {
-      throw new Error('invalid_argument: JWT expiresIn is not a number')
-    }
-  }
+  let payloadResult: Partial<JWTPayload> = payload;
 
-  const did = issuers[0].issuer;
-  const fullPayload: Partial<JWTPayload> = { ...timestamps, ...payload, iss: did }
-
-  let payloadResult = encodeSection(fullPayload, canonicalize);
-  let jws = "";
+  let jwt = "";
   for (let i = 0; i < issuers.length; i++) {
     const issuer = issuers[i];
-    if (!issuer.signer) throw new Error('missing_signer: No Signer functionality has been configured')
-    if (!issuer.issuer) throw new Error('missing_issuer: No issuing DID has been configured')
-    if (issuer.issuer !== did) throw new Error('invalid_argument: all issuers must be the same DID')
-    if (!issuer.alg) throw new Error('missing_alg: No signing algorithm has been configured')
+
```

Comment:
we do need to check all the issuers are the same. otherwise it wont work for Verifiable Conditions

Reply:
it get checked in the jwt-vc
 ``    for (const iss of issuer) {
      if (iss.did !== did) {
        throw new Error('All issuers must be the same did to comply with the Verifiable Conditions spec');
      }``